plugins/modules/sudoers.py

```diff
@@ -292,6 +292,7 @@ def main():
     module = AnsibleModule(
         argument_spec=argument_spec,
         mutually_exclusive=[['user', 'group']],
+        required_one_of=[['user', 'group']],
         supports_check_mode=True,
         required_if=[('state', 'present', ['commands'])],
```

Comment:
```suggestion
        required_if=[
            ('state', 'present', ['commands']),
            ('state', 'present', ['user', 'group'], True),
        ],
```
(The `True` makes sure that only one of them is required.)